projects/canopy/src/lib/separator/docs/guide.stories.mdx

```diff
@@ -0,0 +1,62 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Separator/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Separator
+
+<p class="standfirst">Separators are used to visually divide sections of content.</p>
+
+![example](docs/accordion/example.png)
+
+## Usage
+
+Use this component when you want to split your content into clear sections or groupings.
+
+Import the separator module into your module:
+
+```
```

Comment:
Could you add `js` to this code snippet please?

Reply:
done 👍